samples/nrf9160/lwm2m_client/overlay-assist-cell.conf

```diff
@@ -7,5 +7,4 @@ CONFIG_LWM2M_VERSION_1_1=y
 CONFIG_BASE64=y
```

Comment:
Can we remove CONFIG_BASE64 , CONFIG_LWM2M_RW_SENML_JSON_SUPPORT and   CONFIG_LWM2M_SECONDS_TO_UPDATE_EARLY from those agps and cell assintans overlays?

Reply:
Will do that in another PR.